Admin/AbstractAdmin.php

```diff
@@ -2873,7 +2873,8 @@ public function getDashboardActions()
     final public function getSearchResultLink($object)
     {
         foreach ($this->searchResultActions as $action) {
-            if ($this->hasRoute($action) && $this->isGranted(strtoupper($action), $object)) {
+            $role = 'show' === $action ? 'VIEW' : strtoupper($action);
```

Comment:
https://github.com/sonata-project/SonataAdminBundle/commit/6b367ee9860d4c05fbb3656882188a747ba756f1

This is on 3.x. :-)

I will check that.